Microsoft.Toolkit.Uwp.UI.Controls/InAppNotification/Styles/MSEdgeNotificationStyle.xaml

```diff
@@ -20,11 +20,16 @@
         </ResourceDictionary>
     </ResourceDictionary.ThemeDictionaries>
 
+    <x:Double x:Key="SystemControlMSEdgeNotificationDismissButtonSize">40</x:Double>
+    <Thickness x:Key="SystemControlMSEdgeNotificationDismissButtonMargin">24,0,0,0</Thickness>
+    <x:Double x:Key="SystemControlMSEdgeNotificationDismissButtonTranslate">18</x:Double>
+    <VerticalAlignment x:Key="SystemControlMSEdgeNotificationDismissButtonVerticalAlignment">Center</VerticalAlignment>
+    
```

Comment:
Maybe we could take the opportunity to also have a dedicated static resource for the hardcoded `BorderThickness="2"` from the `<Border x:Name="TextBorder">` below?

Reply:
I've added `SystemControlMSEdgeNotificationButtonBorderThickness`.